std-bits/table/src/main/java/org/enso/table/data/column/storage/StringStorage.java

```diff
@@ -18,8 +20,8 @@
 
 /** A column storing strings. */
 public final class StringStorage extends SpecializedStorage<String> {
-
   private final TextType type;
+  private long _countLeadingTrailingWhitespace = -1;
```

Comment:
is the leading `_` a convention? I haven't seen it much in our code. It feels like we could just do
```suggestion
  private long countLeadingTrailingWhitespace = -1;
```